addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -315,6 +316,80 @@
             </field>
         </record>
 
+        <!-- Inherited form view for mass mailing's form specifically -->
+        <record model="ir.ui.view" id="mailing_mailing_view_full_width_form">
```

Comment:
mailing_mailing_view_form_full_width ? just to follow guidelines (view_form indicates a form view, then a suffix for details)